Documentation/Getting-Started/ceph-openshift.md

```diff
@@ -21,6 +21,19 @@ oc create -f operator-openshift.yaml
 oc create -f cluster.yaml
 ```
 
+Helm Installation:
+
+Set the `openshift` to `true` in `rook-release/rook-ceph` values.yaml.
```

Comment:
Let's just mention this setting, and then link to the helm chart doc (operator-chart.md). No need to put the same helm commands here too.